stock_available_mrp/models/product_product.py

```diff
@@ -109,8 +109,17 @@ def _compute_available_quantities_dict(self):
                 )
 
             res[product.id]["potential_qty"] = potential_qty
-            res[product.id]["immediately_usable_qty"] = potential_qty
-
+            # Normal BoM products (those which are manufactured) are regular stored
+            # products and their immediately_usable_qty will follow general rules.
+            # Phantom BoM products (kits) don't have real stock so their available
+            # to promise quantity will be the same as the potential.
+            if bom_id.type == "phantom":
+                res[product.id]["immediately_usable_qty"] = potential_qty
+            elif (
+                self.env.company.stock_available_mrp_add_potential
```

Comment:
Is this company always the more suitable one for the computation?

Reply:
BoMs `company_id` field isn't required, this one is the most neutral I came up with. Another option is to set it globally as config parameter.

Reply:
I'm thinking that maybe we could simply have the default behavior of adding potential to available to promise stock with option of overriding it at BoM level. This way, it could be still possible to have a default value at instance level and we don't have to deal with possible company quirks

Reply:
OK for me

Reply:
Default value in BoM should preserve the old behavior

Reply:
Yes. The old behavior was to add them: https://github.com/OCA/stock-logistics-warehouse/blob/12.0/stock_available_mrp/models/product_product.py#L123